install/kubernetes/cilium/values.yaml

```diff
@@ -1576,3 +1576,6 @@ clustermesh:
 externalWorkloads:
   # -- Enable support for external workloads, such as VMs (false by default).
   enabled: false
+
+# -- Configure if conflicting routes are blacklisted (true by default).
+blacklistConflictingRoutes: true
```

Comment:
We moving from using blacklist/whitelist to denylist/allowlist. Since the underlying Cilium option is named `blacklist-conflicting-routes` and unlikely to change at this point, I think `blacklistConflictingRoutes` make sense for the Helm value. However, maybe the documentation comment could use the denylist instead? cc @joestringer

Reply:
Definitely no problem. I'm a bit unsure about the wording though. Should I write "Configure if conflicting routes are denylisted" or "Configure if conflicting routes are put on the denylist"?